time_tracker_jsnjack@gmail.com/extension.js

```diff
@@ -26,7 +26,11 @@ const TTNotificationBanner = new Lang.Class({
     addAction: function(label, callback) {
         // Style buttons
         var extra_style = "";
-        if (label === restart_button_name) {
+        
+        if (label === preferences_button_name) {
+            label = _("Preferences");
```

Comment:
It should be, yes. But It don't work without my modification. This is not elegant but it is work, and don't change anything for the program.

I have think to change lines 16, 17, 18 to remove name of wichs are double like 
preferences_button_name = _("Preferences"),
to
preferences_button_name, (line 16)